app/views/account/preferences/_appearance.html.erb

```diff
@@ -55,4 +55,15 @@
   <%= f.number_field(:layout_count, class: "form-control", min: 1) %>
 </div>
 
+<div class="form-group mt-3 form-inline">
+  <%= f.label(:image_size, :prefs_image_size.t) %>
+  <%= f.select(:image_size,
+               options_for_select(
+                 User.image_sizes.filter_map { |key, value|
+                   [key.titleize, User.image_sizes.key(value)] if (value >= 3)
```

Comment:
Maybe I'm misunderstanding, but it looks like this is not translating anything?  It looks like it is printing the internal key representation in the dropdown.  That really needs to be protected, in case for any reason we have to change the data representation.  Also can we do something other than "3"? Lastly, isn't User.image_sizes.key(value) just == key?

```
["image_show_#{key}".to_sym.l, key] if value > User.image_sizes[:small]
```

Reply:
oh yeah. Forgot to translate this...

Reply:
Yeah that last one was absurd. I've got a cold

Reply:
Thanks for that rewrite.